dsp/src/iir.rs

```diff
@@ -11,8 +12,7 @@ use core::f32;
 /// To represent the IIR coefficients, this contains the feed-forward
 /// coefficients (b0, b1, b2) followd by the negated feed-back coefficients
 /// (-a1, -a2), all five normalized such that a0 = 1.
-#[derive(Copy, Clone, Default, Deserialize, Serialize)]
-pub struct Vec5(pub [f32; 5]);
+pub type Vec5 = [f32; 5];
```

Comment:
The reason I wanted a newtype is to be able to implement things on it, like in `irr_int`.

Reply:
This type isn't supported by serde-json-core and it was why we were getting panics previously. We can't have anonymous fields in structs, or we will panic when deserializing them. If we want a custom type, we have to implement it with named fields.

That being said, perhaps we can redefine `Vec5` to
```rust
pub struct Vec5 {
    data: [f32; 5],
}
```

to get the best of both worlds?

Reply:
Yes. That's fine!

Reply:
In fact, maybe a trait for the type alias is better.

Reply:
#274